app/rss.py

```diff
@@ -0,0 +1,62 @@
+import xml.etree.ElementTree as ET
+
+from pathlib import Path
+from podgen import Podcast, Episode, Person, Media
+
+from flask import Response
+from app import app
+
+from . import models
+
+file = Path('app/templates/feed_template.xml')
+
+
+class RssResponse(Response):
+    def __init__(self, response, **kwargs):
+        if 'mimetype' not in kwargs and 'contenttype' not in kwargs:
+            if response.startswith('<?xml'):
+                kwargs['mimetype'] = 'application/xml'
+        return super().__init__(response, **kwargs)
+
+
+class RssPodcast(Podcast):
+    """
+    Custom class for comfortable work with the Podcast
```

Comment:
There needs to be a **very** good justification to make your own implementation of something as standardized as  RSS feeds.